cloudpathlib/cloudpath.py

```diff
@@ -658,7 +657,7 @@ def _upload_local_to_cloud(self, force_overwrite_to_cloud: bool = False):
         # if cloud does not exist or local is newer or we are overwriting, do the upload
         if (
             not self.exists()  # cloud does not exist
-            or (self._local.stat().st_mtime > self.stat().st_mtime)
+            or (self._local.stat().st_mtime >= self.stat().st_mtime)
```

Comment:
@pjbull I was getting a test failure because two writes were happening so fast that they produced the same timestamp. Let me know what you think about this. Alternatively, we could add a sleep to the test, but I think it's a genuine situation that could come up in the wild too.

Reply:
Nevermind, I think this was fixed by #112. This is changed back to the way it was before.